### PR TITLE
feat: enable metrics/traces collection from instrumentation by default

### PR DESCRIPTION
**Pull request by mat-rumian**
##### Description

It's time to enable tracing ingestion by default. OTC will be deployed with SKC by default.

---

##### Checklist

Remove items which don't apply to your PR.

- [x] Changelog updated

###### Testing performed

- [x] Deploy chart to cluster

### CODE REVIEW

File: deploy/helm/sumologic/values.yaml [hunk 1]
```diff
@@ -252,9 +252,9 @@ sumologic:
         failureThreshold: 3
 
   ### Traces configuration
-  ## Set the enabled flag to true to enable traces ingestion.
+  ## Set the enabled flag to false to disable traces ingestion.
   traces:
-    enabled: false
+    enabled: true
```
Comment (sumo-drosiek) [inline deploy/helm/sumologic/values.yaml:257]: @SumoLogic/open-source-collection-team WDYT?

File: deploy/helm/sumologic/values.yaml [hunk 2]
```diff
@@ -177,7 +177,7 @@ sumologic:
           name: traces
           config-name: endpoint-traces
           properties:
-            content_type: Zipkin
+            content_type: OtlpHttpProto
```
Comment (perk-sumo) [inline deploy/helm/sumologic/values.yaml:180]: Is that a breaking change? We need to make sure it's not if that is to be included in v2.

**Review by perk-sumo — CHANGES_REQUESTED** [review]
Hey, before we can proceed with this need confirmation that this will not break current users who have tracing enabled.

**Comment by mat-rumian** [comment]
As we decided - PR postponed till 3.0

File: deploy/helm/sumologic/values.yaml (hunk 2 above)
Comment (perk-sumo) [reply deploy/helm/sumologic/values.yaml:207]: Hey @mat-rumian should we call that change out in the changelog as well?

Comment (mat-rumian) [reply deploy/helm/sumologic/values.yaml]: Hey, I've reverted the change of the content_type.

**Review by swiatekm-sumo — APPROVED** [review]
LGTM

**Pull request merged**